x/gov/init.go

```diff
@@ -95,6 +69,34 @@ func (*Initializer) FromGenesis(opts weave.Options, db weave.KVStore) error {
 			return err
 		}
 	}
+
+	// handle election rules
+	rulesBucket := NewElectionRulesBucket()
+	for i, r := range governance.Rules {
+		electorateID := encodeSequence(r.ElectorateID)
+		_, _, err := electBucket.GetLatestVersion(db, electorateID)
```

Comment:
The electorate is validated for an existance but the result (latest available version) is not used anywhere. Are we not versioning the data anymore?

Reply:
For init, we just make sure that the reference exists.
The rules points to the ID, and when we create a proposal, we then `GetLatestVersion` *at that time* not at the time of rule construction.